posthog/api/csp.py

```diff
@@ -90,12 +90,14 @@ def parse_report_to(data: dict) -> dict:
 
 
 def build_csp_event(props: dict, distinct_id: str, session_id: str, version: str, user_agent: Optional[str]) -> dict:
+    props = {f"$csp_{k}": v for k, v in props.items() if k not in ["$current_url"]}
```

Comment:
This removes `$current_url` from `props` rather than simply not adding the prefix. Is this intended? Shouldnt you guarantee `$current_url` is in it?

Reply:
I've changed it to make it clearer, so we're no longer handling that with the other report properties.